src/libAtomVM/bif.c

```diff
@@ -108,6 +108,48 @@ term bif_erlang_bit_size_1(Context *ctx, uint32_t fail_label, int live, term arg
     return term_from_int32(len);
 }
 
+bool get_sub_binary_slice(term bin_term, term pos_term, term len_term, struct SubBinarySlice *slice)
```

Comment:
The name didn't suggest me the purpose of this function without looking at the actual implementation. What about `maybe_get_sub_binary_slice` or even `sanitize_sub_binary_slice`.
Furthermore,
1. we keep prefixing "public" function names, so a "public" function declared in `bif.h` is named such as `bif_sanitize_sub_binary_slice`.
2. I would keep this function in term.h, since it is a common helper between nifs and bifs.

Reply:
It might be even a more reusable API if take indexes as avm_int_t, so we might use it internally in other situations.